app/views/ui-components/bs4/v1/navs/_header.html.erb

```diff
@@ -1,7 +1,7 @@
 <hidden id='dchbx_enroll_date_of_record' class='d-none'><%= TimeKeeper.date_of_record.iso8601 %></hidden>
 <header class="bg-white navbar-primary py-2 <%= 'pre_prod_nav_color' if ENV['ENROLL_REVIEW_ENVIRONMENT'] == true %>">
 
-  <nav class="navbar navbar-default navbar-expand-lg container py-1">
+  <nav class="navbar navbar-default navbar-expand-lg container py-1 px-0">
```

Comment:
rather than `py-1 px-0` you can do `p-0` which will take care of both vertical and horizontal padding

Reply:
so we're fine with removing the vertical padding then entirely? that would make the y padding 0 instead of 1

Reply:
I misread sorry! fine as is